test/sdk/restapi/http-api-test.go.nolint

```diff
@@ -154,7 +154,7 @@ func main() {
 
 		if count, _, err := alphaCli.SDKApi.GetPlayerCount(ctx); err != nil {
 			log.Fatalf("Error retrieving player count: %s", err)
-		} else if count.Count != "" { // "" is 0, which is empty. This is what the generated client gives us.
+		} else if count.Count != "0" {
```

Comment:
🤔 does this file still need to be `.nolint` or could we leave it now as a `.go` file?

Reply:
I added an exclusion for golang-ci in other PR.
https://github.com/googleforgames/agones/blob/main/.golangci.yml#L26-L27

I'll update to modify this file to `.go` file, because lint also excudes this file.

Reply:
Renamed from `http-api-test.go.nolint` to `http-api-test.go`